ipa-core/src/bin/report_collector.rs

```diff
@@ -53,8 +60,20 @@ struct Args {
     #[clap(flatten)]
     input: CommandInput,
 
+    /// The encrypted input for H1
+    #[arg(long, value_name = "H1_ENCRYPTED_INPUT_FILE")]
```

Comment:
I suggest a different approach - these parameters seem to be used only for `OprfIpa`, so it may make sense to move them insode `ReportCollectorCommand` enum. That way you can make them non-optional and get rid of `as_ref` calls when you actually want to get the value out